### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by theSecondBlueWizard** [Nabla-NTNU/nablaweb-vue#39]
Would be nice to do a ground-level look at where we conform and where we can make sure we comply with accessibility guidelines. At least a skip-link should be provided.

https://vuejs.org/guide/best-practices/accessibility.html
